src/prompts/plugin.ts

```diff
@@ -1,7 +1,7 @@
 import { fetch } from 'undici';
 import { getLang } from '../utilities/getLang.js';
 
-function upperCase(string) {
+function upperCase(string: string) {
```

Comment:
```suggestion
function upperCase(string: string | null) {
```

And in 
https://github.com/Tofix26/sern-cli-ts-rewrite/blob/e0dd4af7f2ee609aa02f0ef07a461119bd3e6752/src/utilities/getLang.ts#L4-L8

Make it return type `Promise<string>` and you can remove the jsdoc type